lib/course_planner_web/templates/layout/supervisor_app_navbar.html.eex

```diff
@@ -0,0 +1,23 @@
+<%= CoursePlannerWeb.SharedView.navbar "Course Planner" do %>
```

Comment:
is there a difference between coordinator and supervisor's navbar ?

Reply:
Nope, though I might need to customize it so made a copy.